src/api/collection.ts

```diff
@@ -64,6 +64,28 @@ export class API extends HubAPI {
     }));
   }
 
+  getPublishedCount(repositoryPath: string) {
+    return new Promise((resolve, reject) => {
+      this.http
+        .get(`v3/plugin/ansible/content/${repositoryPath}/collections/index/`)
+        .then((result) => {
+          resolve(result.data.meta.count);
+        })
+        .catch((err) => reject(err));
+    });
```

Comment:
this whole bit is equivalent to

```suggestion
    return this.http
      .get(`v3/plugin/ansible/content/${repositoryPath}/collections/index/`)
      .then((result) => result.data.meta.count);
```